### PR TITLE
list sms method added

### PR DESCRIPTION
**Pull request by sokalauvalery**

### CODE REVIEW

File: lib/messagebird/client.rb [hunk 1]
```diff
@@ -232,6 +232,13 @@ def message(id)
       Message.new(request(:get, "messages/#{id}"))
     end
 
+    # Retrieve messages with optional paging and status filter.
+    def message_list(limit: 10, offset: 0, status: '')
+      query = "messages?limit=#{limit}&offset=#{offset}"
+      query = query + '&status=' + status if status != ''
```
Comment (Funfun) [inline lib/messagebird/client.rb]: Please use https://apidock.com/ruby/URI/encode_www_form/class

Comment (sokalauvalery) [reply lib/messagebird/client.rb]: Good point! Thank you! fixed.

File: lib/messagebird/client.rb [hunk 2]
```diff
@@ -232,6 +232,16 @@ def message(id)
       Message.new(request(:get, "messages/#{id}"))
     end
 
+    # Retrieve messages with optional paging and status filter.
+    def message_list(limit: 10, offset: 0, status: '')
+      params = [['limit', limit], ['offset', offset]]
```
Comment (Funfun) [inline lib/messagebird/client.rb]: the hash variant would be readable thou

Comment (Funfun) [reply lib/messagebird/client.rb]: i missed one thing, please use options hash instead of keyword argument, as ruby 1.9.3 has no support for it. Cheers

Comment (sokalauvalery) [reply lib/messagebird/client.rb]: fixed! thank you

**Pull request merged**